RIBs.podspec

```diff
@@ -11,5 +11,5 @@ RIBs is the cross-platform architecture behind many mobile apps at Uber. This ar
   s.source           = { :git => 'https://github.com/uber/RIBs.git', :tag => 'v' + s.version.to_s }
   s.ios.deployment_target = '8.0'
   s.source_files = 'ios/RIBs/Classes/**/*'
-  s.dependency 'RxSwift', '~> 4.5.0'
+  s.dependency 'RxSwift', '~> 5.0.1'
```

Comment:
I think `~> 5.0` is sufficient or else this could easily cause conflicts with other Pods in a project that use, say, 5.0.2 or 5.1 when those versions are released